CharlieBackend.Business/Services/CourseService.cs

```diff
@@ -52,38 +54,39 @@ public async Task<Result<CourseDto>> CreateCourseAsync(CreateCourseDto courseDto
 
         public async Task<Result<IList<CourseDto>>> GetAllCoursesAsync()
         {
-            return Result<IList<CourseDto>>.GetSuccess(_mapper
-                    .Map<IList<CourseDto>>(await _unitOfWork.CourseRepository.GetAllAsync()));
+            var courses = _mapper.Map<List<CourseDto>>(await _unitOfWork.CourseRepository.GetAllAsync());
+
+            return Result<IList<CourseDto>>.GetSuccess(courses);
         }
 
-        public async Task<Result<CourseDto>> UpdateCourseAsync(long id, UpdateCourseDto updateCourseDto)
+        public async Task<Result<CourseDto>> UpdateCourseAsync(long id, UpdateCourseDto courseModel)
```

Comment:
Please put back the variable name "..Dto" instead of "... Model" for this and other models in services and let the name of the variable remain for what it is intended ("update...Dto", "create...Dto" etc)

Reply:
agree with Bohdan

Reply:
okay